proxy/mitmproxy.py

```diff
@@ -37,8 +37,8 @@ def start(self):
 
         return self.process
 
-    def stop(self):
-        if self.mode is "record":
+    def stop(self, kill=True):
+        if kill:
             # Record mode. Send proxy stop command and wait for it to close.
```

Comment:
This comment no longer makes sense if we stop the proxy in replay mode too.

Reply:
removed the `Record mode.`